server/service/src/sync/translations/name.rs

```diff
@@ -230,12 +231,12 @@ impl SyncTranslation for NameTranslation {
                 .or(created_date.map(|date| date.and_hms_opt(0, 0, 0).unwrap())),
             date_of_death,
             custom_data_string,
+            deleted_datetime: existing_name.and_then(|name| name.deleted_datetime),
```

Comment:
Why is this using existing_name? If the name was merged in OG, then this wouldn't populate the deleted datetime at all....

Reply:
and we're not deleting patients in OMS....

Reply:
Yeah you are right it's not needed.